db/migrate/20191017151119_add_external_ipam_id_to_subnets.rb

```diff
@@ -0,0 +1,9 @@
+class AddExternalIpamIdToSubnets < ActiveRecord::Migration[5.2]
+  def up
+    add_column :subnets, :external_ipam_id, :integer
```

Comment:
We do appear have indices for all references (e.g. `t.index ["dns_id"], name: "index_subnets_on_dns_id"`) can you add one too?